x-pack/plugins/security_solution/public/management/components/console/components/console_manager/components/confirm_terminate.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFocusTrap,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+export interface ConfirmTerminateProps {
+  onConfirm: () => void;
+  onCancel: () => void;
+}
+
+export const ConfirmTerminate = memo<ConfirmTerminateProps>(({ onConfirm, onCancel }) => {
+  return (
+    <div
+      className="euiOverlayMask"
+      style={{ position: 'absolute' }}
+      data-test-subj="consolePopupTerminateConfirmModal"
+    >
+      <EuiFocusTrap>
+        <EuiPanel className="terminate-confirm-panel">
+          <EuiCallOut
+            color="primary"
+            iconType="iInCircle"
+            data-test-subj="consolePopupTerminateConfirmMessage"
+            title={
+              <FormattedMessage
+                id="xpack.securitySolution.console.popup.confirmTitle"
+                defaultMessage="Terminate this session"
+              />
+            }
+          >
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.securitySolution.console.popup.terminateMessage"
+                  defaultMessage="This will end your console session. Do you wish to continue?"
+                />
+              </p>
```

Comment:
I think you are right, but only because there is only one `<p>` inside of this `<EuiText>`.  Paragraph tags get additional styles, but looking at the browser tools, looks like it may be stripped from the last `<p>` child. I'll remove it.